Tasks/Maven/task.json

```diff
@@ -16,7 +16,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 64
+        "Patch": 65
```

Comment:
I am also hotfixing Maven (and Gradle) for S105. Ping me so that we don't duplicate the effort.